requirements/basic.txt

```diff
@@ -0,0 +1,12 @@
+# build and load models and data, no web running or plugins
+
+xarray>=0.16.2
+scipy
```

Comment:
I think script could go in `core.txt` as I think it's only used for the mode solver?

Reply:
Yeah we can do that, it is required in xarray so it won't really make a difference, but oh well.

Reply:
ah, right. Well an sent a list of packages that are easy installable on web: https://pyodide.org/en/latest/usage/packages-in-pyodide.html. `scipy` and `xarray` are on there so that's nice.